library/core/src/num/f128.rs

```diff
@@ -755,16 +755,24 @@ impl f128 {
     #[inline]
     #[unstable(feature = "f128", issue = "116909")]
     // #[unstable(feature = "float_minimum_maximum", issue = "91079")]
+    #[rustc_const_unstable(feature = "f128", issue = "116909")]
```

Comment:
Do these need to be added for some reason? I thought they are redundant if an `#[unstable]` function is marked `const`.

Reply:
I got an error telling me to add them, it was similar to [these ones](https://github.com/rust-lang/rust/blob/da1e442ebde8d527f3872c0cac5b371ff6ea0cc7/tests/ui/consts/min_const_fn/recursive_const_stab_unstable_if_unmarked.stderr#L8).

Reply:
They need to be added for functions that use const-unstable language features, including const-unstable intrinsics.

Reply:
Do you still get the error now that you added fallback bodies? AFAIK the error only applies to intrinsics without fallback bodies.

Reply:
Indeed, the error disappear with the fallback bodies, removed the attribute were applicable.